index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="pt-BR">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SSR-Simulator</title>
+
+    <!-- Styles -->
+    <link rel="stylesheet" href="css/styles_reset.css">
+    <link rel="stylesheet" href="css/style.css">
+
+</head>
+
+<body>
+
+    <!-- Header -->
+
+    <header>
+
+        <div class="TopBar" alt="Barra Superior de Navegação">
+
+            <!-- Logo -->
+            <span class="TopLogo">
+                <a href="#" class="CathoLogo" alt="Logotipo da Catho"><img src="imgs/cathoLogo.svg" alt="Logotipo da Catho"></a>
+            </span>
+
+            <!-- Main Navigation Menu -->
+            <span>
+                <nav class="NavigationBar" alt="Menu de Navegação">
+                    <ul class="NavigationMenu">
+                        <li><a href="#">Catho Components</a></li>
+                        <li><a href="#">Quantum</a> </li>
+                    </ul>
+                </nav>
+            </span>
+
+            <!-- Hidden Navigation Menu -->
+            <div>
+                <nav class="HiddenNavigationBar" alt="Barra de Navegação Secundária">
+                    <ul>
+                        <li><a href="#">Fale Conosco</a></li>
+                        <li><a href="#">Conheça</a> </li>
+                        <li><a href="#">Portfolio</a></li>
+                        <li><a href="#">...</a></li>
+                        <li><a href="#">...</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </div>
+    </header>
+
+
+    <!-- Main Content -->
+
+    <main>
+        <div class="MainContent">
+
+        </div>
+
+    </main>
+
+    <!-- Footer -->
+
+    <footer>
+        <div class="FooterInformation">
```

Comment:
how about of put that class in `footer` element?
That way, we dont need this `div` element